commands/update-version-in-file/update_version_in_file_test.go

```diff
@@ -0,0 +1,69 @@
+package updateversioninfile
+
+import (
+	"github.com/stretchr/testify/require"
+	"regexp"
+	"testing"
+)
+
+func TestNoMatchingPatternFoundReturnsIdenticalFile(t *testing.T) {
+	replacementStr := "KURTOSIS_CORE_VERSION: string = \"0.1.3\""
+	searchPatternStr := "KURTOSIS_CORE_VERSION: string = \"[0-9A-Za-z_./-]+\""
```

Comment:
Careful; you've copied in the `[0-9A-Za-z_./-]+` but that means that if we ever change it in the actual code then this will be using an oudated version. This should be dynamic